packages/OreIdWalletConnect/src/ActiveSessionButton/ActiveSessionButton.scss

```diff
@@ -7,6 +7,7 @@
   background: transparent;
   cursor: pointer;
   padding: 10px 12px;
+  white-space: nowrap;
```

Comment:
I believe this style should stay with the "parent" or make the button "inline-block"

Reply:
i disagree